packages/typespec-test/test/todo_non_branded/generated/typespec-ts/src/responses.ts

```diff
@@ -17,13 +17,13 @@ export interface UsersCreate200Response extends HttpResponse {
   body: UserCreatedResponseOutput;
 }
 
-/** The request conflicts with the current state of the server. */
+/** The user already exists */
```

Comment:
Could you help check the relevant pr?

Reply:
Didn't find relevant pr yet, but this seems a bugfix and both apply to js and openapi emitters.

Reply:
> Didn't find relevant pr yet, but this seems a bugfix and both apply to js and openapi emitters.

@v-jiaodi thanks for di's sharing and should be here: https://github.com/microsoft/typespec/pull/4322.